recipes-extended/jailhouse/jailhouse_0.11.bb

```diff
@@ -0,0 +1,121 @@
+SUMMARY = "Jailhouse, i.MX fork"
+HOMEPAGE = "https://github.com/siemens/jailhouse"
+SECTION = "jailhouse"
+LICENSE = "GPL-2.0 & BSD-2-Clause"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=9fa7f895f96bde2d47fd5b7d95b6ba4d \
+                 file://tools/root-cell-config.c.tmpl;beginline=6;endline=33;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/jailhouse/hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/jailhouse/cell-config.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/arm/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/arm64/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/x86/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://driver/jailhouse.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+"
+
+SRCBRANCH = "lf-5.4.y"
+IMX_JAILHOUSE_SRC = "git://source.codeaurora.org/external/imx/imx-jailhouse.git;protocol=ssh"
+
+SRC_URI = "${IMX_JAILHOUSE_SRC};branch=${SRCBRANCH} \
+           file://0001-scripts-tools-switch-to-python3.patch \
+"
```

Comment:
Defining `IMX_JAILHOUSE_SRC` unconditional and using it in adjacent `SRC_URI` is a bit pointless IMHO, as it cannot be overridden.

It is maybe easier to do one of the following:
1. Define `IMX_JAILHOUSE_SRC` conditional and default to `codeaurora.org`
2. Include `codeaurora.org` directly into `SRC_URI` and drop `IMX_JAILHOUSE_SRC` completely

Reply:
Okay, np, we can use `codeaurora.org` directly into `SRC_URI`